packages/core/src/docdb/commands/addRegion.ts

```diff
@@ -17,22 +18,38 @@ import { CreateGlobalClusterWizard } from '../wizards/createGlobalClusterWizard'
 import { CreateDBClusterMessage } from '@aws-sdk/client-docdb'
 import { createInstancesForCluster } from './createCluster'
 
-export async function addRegion(node: DBClusterNode): Promise<void> {
+export async function addRegion(node: DBClusterNode | DBGlobalClusterNode): Promise<void> {
     if (!node) {
         throw new ToolkitError('No node specified for AddRegion')
     }
 
     return telemetry.docdb_addRegion.run(async () => {
-        if (node.clusterRole !== 'regional') {
-            void vscode.window.showErrorMessage('Currently supported for standalone clusters only.')
-            return
-        }
+        let globalClusterName = undefined
 
-        if (node.cluster.DBClusterMembers?.length === 0) {
-            void vscode.window.showErrorMessage(
-                localize('AWS.docdb.addRegion.noInstances', 'Cluster must have at least one instance to add a region')
-            )
-            throw new ToolkitError('Cluster must have at least one instance to add a region', { cancelled: true })
+        if (node instanceof DBClusterNode) {
+            if (node.clusterRole !== 'regional') {
+                void vscode.window.showErrorMessage('Currently supported for standalone clusters only.')
+                return
```

Comment:
Should this also throw an error like the other two situations?

Reply:
No, this was there as a warning only.  Now that I've updated the menu expressions, the command should only be available for regional clusters.  Left it in just in case.